requirements/dev.txt

```diff
@@ -10,7 +10,7 @@
 --no-binary psycopg2
 
 git+https://github.com/zulip/django-bitfield@0d2b15cdb5af5ddec88d41cac19c0f2ce1b1ad38#egg=django-bitfield==1.9.3+dev.0d2b15cdb5af5ddec88d41cac19c0f2ce1b1ad38
-git+https://github.com/python/mypy.git@6a934d32124991a77ea433856cf2d62871ceb632#egg=mypy==0.610+dev.6a934d32124991a77ea433856cf2d62871ceb632
+git+https://github.com/python/mypy.git@6a934d32124991a77ea433856cf2d62871ceb632#egg=mypy==0.620+dev.6a934d32124991a77ea433856cf2d62871ceb632
```

Comment:
These should be able to replace with just `mypy == 0.620`.  We only had to do a commit ID before because 0.610 had a bug we needed fixed.